src/main/java/uk/gov/hmcts/darts/transcriptions/exception/TranscriptionApiError.java

```diff
@@ -93,6 +93,21 @@ public enum TranscriptionApiError implements DartsApiError {
         HttpStatus.CONFLICT,
         TranscriptionsTitleErrors.TRANSCRIPTION_DOCUMENT_ALREADY_HIDDEN.getValue()
     ),
+    TRANSCRIPTION_DOCUMENT_DELETE_NOT_SUPPORTED(
+        TranscriptionsErrorCode.TRANSCRIPTION_DOCUMENT_DELETE_NOT_SUPPORTED.getValue(),
+        HttpStatus.CONFLICT,
+        TranscriptionsTitleErrors.TRANSCRIPTION_DOCUMENT_DELETE_NOT_SUPPORTED.getValue()
+    ),
+    TRANSCRIPTION_DOCUMENT_DELETION_ALREADY_APPROVED(
+        TranscriptionsErrorCode.TRANSCRIPTION_DOCUMENT_DELETION_ALREADY_APPROVED.getValue(),
+        HttpStatus.CONFLICT,
+        TranscriptionsTitleErrors.TRANSCRIPTION_DOCUMENT_DELETION_ALREADY_APPROVED.getValue()
+    ),
+    TRANSCRIPTION_DOCUMENT_DELETION_CAN_NOT_APPROVE_OWN_REQUEST(
+        TranscriptionsErrorCode.TRANSCRIPTION_DOCUMENT_DELETION_CAN_NOT_APPROVE_OWN_REQUEST.getValue(),
+        HttpStatus.BAD_REQUEST,
```

Comment:
Should be a 422 UNPROCESSABLE_ENTITy

Reply:
LLD requirements to throw 400

Reply:
I spoke to @jackmaloney a few weeks ago around 400 vs 422 and I believe the agreement was that future exception handling would be 422 for business validation rules (But legacy ones would not be updated).

As such I suspect this should have been updated to a 422

Reply:
Updated to 422 after confirmation.